dbus/methods.go

```diff
@@ -288,6 +288,15 @@ func (c *Conn) listUnitsInternal(f storeFunc) ([]UnitStatus, error) {
 	return status, nil
 }
 
+// GetUnitByPID returns an array with all currently loaded units. Note that
+func (c *Conn) GetUnitByPID(pid int) (dbus.ObjectPath, error) {
```

Comment:
Input parameter should be a `uint32` to match DBus signature.

Reply:
I agree. I was thinking of using `int` because its what the go stdlib uses for pids (oddly enough), but I'd prefer to adhere to the dbus signature